### ISSUE TITLE
Bug when setting reviewers and review type

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/direct-app#293]
When editing a challenge, you can't update the reviewer.  The review type also always shows community review, even when it's set to internal.

![image](https://user-images.githubusercontent.com/3812270/28972322-467fab2a-78fd-11e7-8f28-8ef41a25c2f0.png)

**Comment by skyhit** [comment]
@ajefts is it because of active challenge?

**Comment by ajefts** [comment]
@skyhit that challenge was active, but shouldn't we still be able to edit the reviewer and review type if the Review phase has not yet started and there are not reviewers set yet?

**Comment by skyhit** [comment]
@ajefts yes for that case, it is possible, so you want to do like that way?

**Comment by ajefts** [comment]
yes, please
